### PR TITLE
Chapter 3: Central limit theorem notebook

### PR DESCRIPTION
**Pull request by aalvar23**

### CODE REVIEW

**Review by bsipocz — CHANGES_REQUESTED** [review]
Please also add this to the chapter3/README.md file so the rendered notebook will be linked from the readme page and show up in the TOC.

File: chapter3/README.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ maxdepth: 1
 ---
 
 astroml_chapter3_Univariate_Distribution_Functions
-```
\ No newline at end of file
+astroml_chapter3_Univariate_Distribution_Functions
```
Comment (bsipocz) [inline chapter3/README.md]: please use the name of the new notebook rather than duplicate the previous one. Thanks!

**Comment by bsipocz** [comment]
Thanks @aalvar23! It looks all good to me (pending I fix the LaTeX rendering issue), but before merging I'll ask @stephenportillo to sign off on the content, too.

**Comment by stephenportillo** [comment]
A couple text changes:
"for the practice of repeated measurements in order to improve the accuracy of the final result" -> "for the practice of repeating measurements to improve the accuracy of the final result"
"Our constraints of" -> "Our constraints on"

It would be nice to demonstrate the CLT on a second distribution to reinforce the fact that it works for arbitrary distributions. In particular, I think it would be useful to show it working on a distribution with fat tails (but still finite standard deviation), like the Laplace distribution.

**Comment by aalvar23** [comment]
I uploaded the bivariate and multivariate notebook to my forked repository and it's showing up on this pull request; I tried to open a new one, but it doesn't give me the option. I checked this last notebook with Stephen though if you wanted to merge both at the same time. If not, I can try to figure out something else.

**Comment by bsipocz** [comment]
yes, it shows up as the PR is from your main branch rather than from separate feature branches.

But now having them both in this PR fine, I want to fix the LaTeX rendering issue before merging this, so merging them both at the same time will be fine.

**Comment by bsipocz** [comment]
I've rebased this, so CI should pick up the fix for the LaTeX rendering.

**Comment by bsipocz** [comment]
I've removed the commit that removes the outputs, I think I have told you that empty notebooks are better, but we may need to keep the outputs in (pending a decision in #39).

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks @aalvar23!

**Pull request merged**